src/background.js

```diff
@@ -9,7 +9,8 @@ chrome.tabs.onUpdated.addListener((tabId, changeInfo, tab) => {
 });
 
 chrome.runtime.onInstalled.addListener(function (object) {
-    let externalUrl = "https://github.com/ranashreyas/nbgitpuller-link-generator-webextension/blob/popout/usage.md";
+    // let externalUrl = "https://github.com/ranashreyas/nbgitpuller-link-generator-webextension/blob/popout/usage.md";
+    let externalUrl = "https://github.com/berkeley-dsep-infra/nbgitpuller-link-generator-webextension/blob/main/usage.md"
```

Comment:
changed on-install usage.md to what it would be when merged. Note, this link would not work right now, as https://github.com/berkeley-dsep-infra/nbgitpuller-link-generator-webextension/blob/main/usage.md does not exist right now